.github/pull_request_template.md

```diff
@@ -8,7 +8,7 @@ Before submitting, please take the time to check the points below and provide so
 * [ ] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
 * [ ] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
 * [ ] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
-* [ ] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) (if applicable)
+* [ ] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) - Pull Requests that are not linked to an issue with you as an assignee will be closed.
```

Comment:
Does this also apply to minor pull requests like fixing typos and grammar in documentation and comments?

People can not self-assignee to an issue, so fixing one typo will be come an involved process of
- opening a bug issue (what else is a typo?), showing the erroneous line of code/text
- waiting for getting the issue assigned
- fixing the typo
- opening a pull request
or
- opening a bug issue (what else is a typo?), showing the erroneous line of code/text
- a maintainer fixing the typo himself and then closing the issue

Reply:
Agreed, I will change that accordingly